cmd/gitops/add/profiles/cmd.go

```diff
@@ -61,8 +59,6 @@ func AddCommand() *cobra.Command {
 
 func addProfileCmdRunE() func(*cobra.Command, []string) error {
 	return func(cmd *cobra.Command, args []string) error {
-		rand.Seed(time.Now().UnixNano())
```

Comment:
This is needed for the math randomizer. Why remove it?

Reply:
This isn't needed anymore because we switched to using UUIDs instead, for now: https://github.com/weaveworks/weave-gitops/pull/1360#discussion_r798409841 :)

Reply:
ah ok!